src/com/lftechnology/java/training/alina/jdbc/service/UserService.java

```diff
@@ -0,0 +1,324 @@
+     * @param result
+     *            {@link ResultSet}
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public void getEmployeeRole(Scanner scanner, Employee employee) throws SQLException {
+        if (employee.getRole().equals(EmployeeRole.ADMIN.role)) {
+            getAdminRole(scanner, employee);
+        } else {
+            getNormalUserRole(scanner, employee);
+        }
+    }
+
+    /**
+     * Gets Admin role functions
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param result
+     *            {@link ResultSet}
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    private void getAdminRole(Scanner scanner, Employee employee) throws SQLException {
+        char choice = ' ';
+        while (choice != Constants.ADMIN_EXIT) {
+            AdminView.displayAdminRoleMenu(employee);
+            choice = UtilityService.getSelectedMenu(scanner, "Select an option (a-f) : ");
+            getAdminRoleOptions(scanner, choice);
+        }
+    }
+
+    /**
+     * Gets option menu for Admin Role
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param option
+     *            {@link Character}
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    private void getAdminRoleOptions(Scanner scanner, char option) throws SQLException {
+        switch (option) {
+        case 'a':
+            LOGGER.log(Level.INFO, "\n========================\nAdd New Employee : \n========================\n");
+            EmployeeController.addNewEmployee(scanner);
+            break;
+        case 'b':
+            LOGGER.log(Level.INFO, "\n========================\nDelete an Employee : \n========================\n");
+            EmployeeController.deleteExistingEmployee(scanner);
+            return;
+        case 'c':
+            LOGGER.log(Level.INFO, "\n========================\nTerminate an Employee : \n========================\n");
+            EmployeeController.terminateExistingEmployee(scanner);
+            return;
+        case 'd':
+            LOGGER.log(Level.INFO, "\n========================\nView Employee List : \n========================\n");
+            EmployeeController.getEmployeeList();
+            return;
+        case 'e':
+            LOGGER.log(Level.INFO, "\n========================\nSearch an Employee : \n========================\n");
+            EmployeeController.searchExistingEmployee(scanner);
+            return;
+        case 'f':
+            LOGGER.log(Level.INFO, "\n========================\nUser successfully logged out.\n========================\n");
+            UserDaoImpl userDao = new UserDaoImpl();
+            userDao.checkEmployeeLogin(scanner);
+        default:
+            LOGGER.log(Level.INFO, "\n========================\nInvalid entry, Please choose from menu option.\n========================\n");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\n========================\nInvalid entry, Please choose from menu option.\n========================\n" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)